Sofa/Component/Collision/Geometry/src/sofa/component/collision/geometry/CubeModel.cpp

```diff
@@ -167,9 +167,9 @@ void CubeCollisionModel::draw(const core::visual::VisualParams* vparams)
     if (!isActive() || !((getNext()==nullptr)?vparams->displayFlags().getShowCollisionModels():vparams->displayFlags().getShowBoundingCollisionModels())) return;
 
     // The deeper in the CubeModel graph, the higher the transparency of the bounding cube lines
-    int level=0;
+    [[maybe_unused]] int level=0;
     CollisionModel* m = getPrevious();
-    float color = 1.0f;
+    [[maybe_unused]] float color = 1.0f;
```

Comment:
Same here. Why keeping it?

Reply:
I will keep this one, as the comment and the code below does not match. I will restore the feature in a next PR